openssl-sys/build/find_normal.rs

```diff
@@ -56,6 +56,15 @@ fn resolve_with_wellknown_homebrew_location(dir: &str) -> Option<PathBuf> {
     None
 }
 
+fn resolve_with_wellknown_pkgsrc_location() -> Option<PathBuf> {
```

Comment:
Is OpenSSL a mandatory install in pkgsrc? It seems like this could false-positive if you have an /opt/pkg but don't have OpenSSL installed through it.

Reply:
OpenSSL is included in the bootstrap package of pkgsrc:

```sh
curl -sSfL https://pkgsrc.joyent.com/packages/Darwin/bootstrap/bootstrap-macos14-trunk-x86_64-20200716.tar.gz | tar tzf - | grep ssl
```

The headers, libraries, `openssl` binary and `openssl.pc` is included. The `pkgin` tool also links to OpenSSL as we can see with `otool -L /opt/pkg/bin/pkgin`:

```
/opt/pkg/bin/pkgin:
[...]
	/opt/pkg/lib/libssl.1.1.dylib (compatibility version 1.1.0, current version 1.1.0)
	/opt/pkg/lib/libcrypto.1.1.dylib (compatibility version 1.1.0, current version 1.1.0)
[...]
```

Should we better check for presence of the following files:

- `/opt/pkg/include/openssl/ssl.h`
- `/opt/pkg/lib/libssl.a`
- `/opt/pkg/lib/libssl.dylib`
- `/opt/pkg/lib/libcrypto.a`
- `/opt/pkg/lib/libcrypto.dylib`

or would it be enough to test for `/opt/pkg/lib/pkgconfig/openssl.pc` and assume the rest is there as well (a `pkg-config` tool is not included with pkgsrc by default)?

Reply:
pkg-config isn't used if it's discovered in a well-known place. Might be best to explicitly look for `/opt/pkg/include/openssl` just in case.